src/components/createEventModal.vue

```diff
@@ -1,27 +1,31 @@
 <template>
-<transition name="fade" mode="out-in">
-  <div class="modal-backdrop">
-    <div class="nav-icons">
-      <router-link to="/discover">
-        <font-awesome-icon icon="arrow-left" class="back-btn fa-2x" color="#1F2845"/>
-      </router-link>
-    </div>
-    <div class="modal-container">
-      <div class="input-box">
-        <div class="input-wrapper">
-          <v-form class="input-form">
-            <v-text-field
-              name="event-title"
-              label="Event Title"
-              class="event-info-input"
-              clearable
-              required
-            ></v-text-field>
-          </v-form>
-        </div>
+  <transition name="fade" mode="out-in">
+    <div class="modal-backdrop">
+      <div class="nav-icons">
+        <router-link to="/discover">
+          <font-awesome-icon
+            icon="arrow-left"
+            class="back-btn fa-2x"
+            color="#1F2845"
+          />
+        </router-link>
       </div>
-      <div class="input-box">
-        <!-- <v-menu
+      <div class="modal-container">
+        <div class="input-box">
+          <div class="input-wrapper">
+            <v-form class="input-form">
+              <v-text-field
+                name="event-title"
+                label="Event Title"
+                class="event-info-input"
+                clearable
+                required
+              ></v-text-field>
+            </v-form>
+          </div>
+        </div>
+        <div class="input-box">
+          <!-- <v-menu
           ref="menu"
```

Comment:
Is there a reason we need the uncommented section still??

Reply:
this is going to be removed tho right?